com.chartboost.helium/Editor/HeliumSKAdNetworkIds.cs

```diff
@@ -0,0 +1,203 @@
+#nullable enable
+#if UNITY_IOS
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Helium;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEditor.iOS.Xcode;
+using UnityEngine;
+using UnityEngine.Networking;
+// ReSharper disable InconsistentNaming
+// ReSharper disable StringLiteralTypo
+// ReSharper disable IdentifierTypo
+// ReSharper disable CommentTypo
+
+namespace Editor
+{
+    /// <summary>
+    /// Adds the SKAdNetwork Ids to the app's Info.plist
+    /// </summary>
+    public class HeliumSKAdNetworkIds
+    {
+        [PostProcessBuild]
+        public static void PostProcess(BuildTarget buildTarget, string pathToBuiltProject)
+        {
+            if (!HeliumSettings.IsSkAdNetworkResolutionEnabled)
+                return;
+            
+            if (buildTarget != BuildTarget.iOS)
+                return;
+
+            var ids = GetSKAdNetworkIds();
+
+            var plistPath = pathToBuiltProject + "/Info.plist";
+            var plist = new PlistDocument();
+            plist.ReadFromFile(plistPath);
+
+            var skanItems = plist.root.CreateArray(SKAdNetworkItemsKey);
+
+            foreach (var skadnetworkID in ids)
+            {
+                var skanId = skanItems.AddDict();
+                skanId.SetString(SKAdNetworkIdKey, skadnetworkID);
+            }
+
+            plist.WriteToFile(plistPath);
+        }
+
+        private const string AppLovin = "https://dash.applovin.com/docs/v1/skadnetwork_ids.json";
```

Comment:
Looks like there are some partner SDKs missing. They should have a json/xml link, right? 🤔

Reply:
Aw.. rip. Yeah, after a while, there are repeats.

Reply:
^ We will have a process to dedup them if needed.